UPGRADE-3.0.md

```diff
@@ -1739,6 +1768,13 @@ UPGRADE FROM 2.x to 3.0
    `Process::setInput()` and `Process::getInput()` that works the same way.
  * `Process::setInput()` and `ProcessBuilder::setInput()` do not accept non-scalar types.
 
+### Monolog Bridge
+
+ * `Symfony\Bridge\Monolog\Logger::emerg()` was removed. Use `emergency()` which is PSR-3 compatible.
+ * `Symfony\Bridge\Monolog\Logger::crit()` was removed. Use `critical()` which is PSR-3 compatible.
+ * `Symfony\Bridge\Monolog\Logger::err()` was removed. Use `error()` which is PSR-3 compatible.
+ * `Symfony\Bridge\Monolog\Logger::warn()` was removed. Use `warning()` which is PSR-3 compatible.
```

Comment:
The deprecations are present in the `2.7` branch too: https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bridge/Monolog/Logger.php#L30